integration_tests/src/main/python/delta_lake_write_test.py

```diff
@@ -44,22 +44,27 @@
                                 "spark.sql.legacy.parquet.datetimeRebaseModeInRead": "CORRECTED",
                                 "spark.sql.legacy.parquet.int96RebaseModeInRead": "CORRECTED"})
 
+def get_writer_with_deletion_vector_property_set(writer, enable_deletion_vectors):
+    return writer.option("delta.enableDeletionVectors", str(enable_deletion_vectors).lower())
+
 def get_last_operation_metrics(path):
     from delta.tables import DeltaTable
     return with_cpu_session(lambda spark: DeltaTable.forPath(spark, path)\
                             .history(1)\
                             .selectExpr("operationMetrics")\
                             .head()[0])
 
-def _create_table(spark, path, schema, partitioned_by=None):
+def _create_table(spark, path, schema, partitioned_by=None, enable_deletion_vectors=False):
```

Comment:
why are we setting it to False by default? Shouldn't we have the params in all of the tests so we know that we are doing the right thing?